### DISCUSSION TITLE
Integration of systray in eww

### DISCUSSION THREAD

**Discussion by carrascomj** [elkowar/eww#225] [Q&A] [answered]
Are there any elegant ways of having a system tray (for `nm-applet`, `cbatticon`, etc.) inside eww? Or is it only possible to do it using something like [`stalonetray`](https://wiki.archlinux.org/title/Stalonetray) or [`trayer`](https://man.archlinux.org/man/extra/trayer/trayer.1.en)? What are the solutions that people are using for systrays with eww?

I am using eww as my bar in a tiling window manager ([leftwm](https://github.com/leftwm/leftwm)), and I would prefer to feed the different applets into an eww box or something that, best-case scenario, disappears when there's none.

**Comment by davidnoronha1** [comment, marked as answer]
There is no elegant way atm, youll have the most luck using stalonetray or trayer, system trays are a messy thing :/
There is an issue about it although no work has been done #111

**Comment by siduck** [comment]
@carrascomj it   has been  added last month, try it!

**Reply by descentcare** [reply]
Is there docs for it? There's no full explanation on https://elkowar.github.io/eww/widgets.html on how to use it, no info on some properties I saw in other people configs.

And it draws its own menus for dropdown on rightclick. I found how to style it just searching issues on github, but there's nothing on it in documentation.

How, for example, would I make a keyboard shortcut to open this menu? I know it might be impossible

**Reply by siduck** [reply]
no idea on opening it with a keybind. its there on the eww bar so i just use my mouse
